pkg/ocm/service_logs.go

```diff
@@ -0,0 +1,9 @@
+package ocm
+
+import (
+	slv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
+)
+
+func (c *Client) ListServiceLogs(clusterId string) (*slv1.ClustersClusterLogsListResponse, error) {
```

Comment:
@renan-campos I think it's worth us making this function take a variadic parameter of `SeverityWarning` that it should search for so that we can make the backend do the work of filtering out events and avoid fetching unnecessary data as at the minute we just pull all events regardless and then iterate over them locally.

So it would be something like:

```
func (c *Client) ListServiceLogs(clusterId string, matchSeverities... ) (*slv1.ClustersClusterLogsListResponse, error)
```

If the user supplies anything, then we use the `Search()` function and only fetch the events matching the given severity. Otherwise, we fetch all events.